README.md

```diff
@@ -1278,6 +1278,14 @@ datetime                              slug     change    status
 2019-07-30 11:00:00+00:00            litex   8.000000  NEWCOMER
 ```
 
+### Available metrics
+
+Gets ahold of a list with all the metrics in sanpy.
```

Comment:
this should be put here:
![image](https://user-images.githubusercontent.com/122794/66394395-ab26c600-e9dd-11e9-9e82-31387afa16f9.png)
instead of the manual bullet list list
Also correct the link from Table of contents - cause it doesn't go here